Sources/PostgresNIO/New/PostgresQuery.swift

```diff
@@ -88,6 +90,26 @@ extension PostgresQuery {
             self.sql.append(contentsOf: "$\(self.binds.count)")
         }
 
+
+        @usableFromInline
+        struct PostgresEncodableSequenceCannotBeEmpty: Error {
+            @usableFromInline init() { }
+        }
+
+        @inlinable
+        public mutating func appendInterpolation<C: Collection>(
+            _ values: C
+        ) throws where C.Element: PostgresThrowingDynamicTypeEncodable {
```

Comment:
You might want to consider constraining to `PostgresArrayEncodable` depending on context